pkgs/development/python-modules/aiohttp-apispec/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder
+, aiohttp, webargs, fetchFromGitHub, callPackage, pytest
+}:
+
+let
+  apispec3 = callPackage ../apispec/3.nix {};
+  jinja2 = callPackage ../jinja2/2.nix {};
+in
```

Comment:
This is sadly not allowed in python-modules because if you have two versions of the same package in one environment python will fall more or less apart. This is a limitation of python, not nix. 

This is no a problem if you move it out of pythonPackages into the package where it is used.

Reply:
Ok. It's no longer in top-level/python-modules so I think it should stay in the directory of the module in case multiple programs need the same old version. Recently when I packed something I discovered an old version of a module I needed that way aswell.

Reply:
The problem is that if we create a python environment with two different versions of jinja and apispec things will fall apart because Python can't handle that. This needs to be done in tribbler itself but the files can be under python-modules as long as they are not references from there.

Reply:
So basically aiohttp-apispec has to get out of python-modules and be part of tribler. Right?

Reply:
Yes